.github/scripts/automergeCrowdinPr.js

```diff
@@ -88,7 +92,9 @@ module.exports = async ({ github, context }) => {
     )
     if (unexpectedFiles.length > 0) {
       console.log(
-        `Files updated in PR #${pr.number} do not match the expectation, please check manually`
+        `The following files updated do not match the expectation: 
+        ${unexpectedFiles.map((file) => file.filename).join(', ')}.
```

Comment:
nit: So it's easier to read?
```suggestion
        ${unexpectedFiles.map((file) => file.filename).join('\n')}.
```